easybuild/easyconfigs/m/MONAI/MONAI-0.9.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'PythonBundle'
+
+name = 'MONAI'
+version = '0.9.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://monai.io/'
+description = """
+MONAI is a PyTorch-based, open-source framework for deep learning in healthcare
+imaging, part of PyTorch Ecosystem.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+github_account = 'Project-MONAI'
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Python', '3.9.5'),
+    ('PyTorch', '1.10.0', versionsuffix),
+    ('einops', '0.3.2'),
+    ('ITK', '5.2.1'),
+    ('NiBabel', '3.2.1'),
+    ('scikit-image', '0.18.3'),
+    ('tensorboard', '2.8.0'),
+    ('torchvision', '0.11.3', versionsuffix),
+    ('tqdm', '4.61.2'),
+]
+
+exts_list = [
+    ('einops', '0.3.2', {
+        'checksums': ['5200e413539f0377f4177ef00dc019968f4177c49b1db3e836c7883df2a5fe2e'],
+    }),
```

Comment:
unnecessary given the dependency on `einops` 
```suggestion
```

Reply:
thanks, I PRd an non-final version of the easyconfig by mistake :slightly_smiling_face: